src/openrct2/sprites.h

```diff
@@ -1121,4 +1121,10 @@ enum
     SPR_IMAGE_LIST_LENGTH = 1000000,
     SPR_IMAGE_LIST_BEGIN = SPR_SCROLLING_TEXT_END,
     SPR_IMAGE_LIST_END = SPR_IMAGE_LIST_BEGIN + SPR_IMAGE_LIST_LENGTH,
+
+    SPR_PAINT_SPLASH_EFFECT_1 = 29014,
```

Comment:
We want to ideally list all of the missing elements i.e. from 29014 -> 29045 are all for splash effect 0.
The sprites are used as follows so you need to name all of them:
```cpp
// Animation of 8 frames for the 4 rotations
constexpr std::array<std::array<uint16_t, 8>, 4> SplashEffect1 = {
    { 29014, 29015, 29016, 29017, 29018, 29019, 29020, 29021 },
    { 29022, ...},
    ...
};
```
```
SPR_SPLASH_EFFECT_1_NORTH_0 = 29014,
SPR_SPLASH_EFFECT_1_NORTH_1
...
SPR_SPLASH_EFFECT_1_NORTH_7
SPR_SPLASH_EFFECT_1_EAST_0
...
```
Also you need to add the entries in the correct part of the sprites.h table not at the end.